### PR TITLE
Remove duplicate completion block definitions

### PR DESCRIPTION
**Pull request by SkylerSeamans**

### CODE REVIEW

File: TeamSnapSDK/SDK/TSDKCompletionBlockTypes.h [hunk 1]
```diff
@@ -69,11 +69,9 @@ typedef void (^TSDKForumSubscriptionArrayCompletionBlock)(BOOL success, BOOL com
 typedef void (^TSDKBroadcastAlertArrayCompletionBlock)(BOOL success, BOOL complete, NSArray <TSDKBroadcastAlert *> *_Nonnull broadcastAlerts, NSError *_Nullable error);
 typedef void (^TSDKAvailabilityGroupCompletionBlock)(BOOL success, BOOL complete, TSDKAvailabilityGroups *_Nullable availabilities, NSError *_Nullable error);
 
-typedef void (^TSDKSimpleCompletionBlock)(BOOL success, NSError *_Nullable error);
```
Comment (larsacus) [inline TeamSnapSDK/SDK/TSDKCompletionBlockTypes.h:72]: Where is the duplicate definition for this? I couldn't find it in a search.

Comment (SkylerSeamans) [reply TeamSnapSDK/SDK/TSDKCompletionBlockTypes.h:72]: Line 14

Comment (larsacus) [reply TeamSnapSDK/SDK/TSDKCompletionBlockTypes.h:72]: ah weird

**Review by larsacus — APPROVED** [review]

**Comment by larsacus** [comment]
When we get new gitlab, hopefully it has repo mirroring and we can get CI setup for this, too

**Pull request merged**